Components/ConnectExport.php

```diff
@@ -97,12 +97,22 @@ public function export(array $ids, ProductStreamsAssignments $streamsAssignments
             }
 
             $connectAttribute = $this->helper->getOrCreateConnectAttributeByModel($model);
+            $excludeInactiveProducts = $this->configComponent->getConfig('excludeInactiveProducts');
+            if ($excludeInactiveProducts && !$model->getActive()) {
+                $connectAttribute->setExportStatus(Attribute::STATUS_INACTIVE);
+                $connectAttribute->setExportMessage(
+                    Shopware()->Snippets()->getNamespace('backend/connect/view/main')->get(
+                        'export/message/error_product_is_not_active',
+                        'Produkt ist inaktiv',
+                        true
+                    )
+                );
+                $this->manager->persist($connectAttribute);
+                $this->manager->flush($connectAttribute);
+                continue;
+            }
 
-            $prefix = $item['title'] ? $item['title'] . ' ('. $item['number'] .'): ' : '';
-            if (empty($item['exportStatus'])
-                || $item['exportStatus'] == Attribute::STATUS_DELETE
-                || $item['exportStatus'] == Attribute::STATUS_ERROR
-            ) {
+            if ($this->helper->isProductExported($connectAttribute)) {
```

Comment:
@sbofirov Shouldn't this be ``if (**!**$this->helper->isProductExported($connectAttribute))`` since if it is already exported the status should be "Update" ? or am I wrong?